initCobraToolbox.m

```diff
@@ -1,4 +1,4 @@
-function initCobraToolbox()
+function [] = initCobraToolbox()
```

Comment:
@laurentheirendt, could you remove the extra `[] = ` in the function signatures?